### DISCUSSION TITLE
How to access image in knowlege repository?

### DISCUSSION THREAD

**Discussion by zjx140** [langgenius/dify#8122] [Help]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing? Tell me about your story.

When I use browser to get image in knowlege repository,it will return code 400:Invalid request. How to access image in knowlege repository?

### 2. Additional context or comments

![image](https://github.com/user-attachments/assets/6cabf3ee-9c74-46d3-b5dc-f4e79129da76)
![image](https://github.com/user-attachments/assets/bf5c4a79-9116-4a64-ab5f-a0a3432aca77)

**Comment by luogenhua** [comment]
172.19.36.11:5555是你的api service的IP和端口吗

**Reply by luogenhua** [reply]
我这里显示的是127.0.0.1:5001   我想把它改成域名，不知道如何修改
